### PR TITLE
Cradio keymap improvements

### PR DESCRIPTION
**Pull request by filterpaper**
Improved cradio default keymap with better indentation and more features.

### CODE REVIEW

File: app/boards/shields/cradio/cradio.keymap [hunk 1]
```diff
@@ -8,95 +8,87 @@
...
+            &kp Q       &kp W  &kp E  &kp R  &kp T         &kp Y    &kp U  &kp I     &kp O    &kp P
+            &kp A       &kp S  &kp D. &kp F  &kp G         &kp H    &kp J  &kp K     &kp L    &kp QUOT
+            &mt LSFT Z  &kp X  &kp C  &kp V  &kp B         &kp N    &kp M  &kp CMMA  &kp DOT  &mt LSFT RET
+                                      &mo 1  &kp LCTL      &kp SPC  &mo 2
+            >;
+        };
+
+        left_layer {
+            bindings = <
+            &kp NUM_1  &kp NUM_2  &kp NUM_3        &kp NUM_4        &kp NUM_5       &kp NUM_6  &kp NUM_7  &kp NUM_8  &kp NUM_9 &kp NUM_0
```
Comment (caksoylar) [inline app/boards/shields/cradio/cradio.keymap:69]: I believe these number keycodes are deprecated; you can run the layers through https://zmk.dev/docs/codes/keymap-upgrader to quickly update them.

Comment (filterpaper) [reply app/boards/shields/cradio/cradio.keymap:69]: Thanks for the pointer.

File: app/boards/shields/cradio/cradio.keymap [hunk 2]
```diff
@@ -1,102 +1,103 @@
...
 #include <dt-bindings/zmk/bt.h>
 
+// Home row mods macro
+#define HRML(k1,k2,k3,k4) &ht LSHFT k1  &ht LALT k2  &ht LCTRL k3  &ht LGUI k4
+#define HRMR(k1,k2,k3,k4) &ht RGUI k1  &ht RCTRL k2  &ht RALT k3  &ht RSHFT k4
 
-&mt {
-  //  flavor = "tap-preferred";
-   // tapping_term_ms = <200>;
+#define ________ &trans
```
Comment (petejohanson) [inline app/boards/shields/cradio/cradio.keymap:12]: TBH, I'm not loving using this... We don't do this everywhere, and it hides what is really going on for these...

**Review by petejohanson — COMMENTED** [review]
One high level point for discussion.

File: app/boards/shields/cradio/cradio.keymap (hunk 2 above)
Comment (filterpaper) [reply app/boards/shields/cradio/cradio.keymap:12]: Good point—overwrite removed as suggested.

Comment (petejohanson) [reply app/boards/shields/cradio/cradio.keymap:12]: Yes, exactly.

File: app/boards/shields/cradio/cradio.keymap [hunk 3]
```diff
@@ -1,102 +1,98 @@
...
+#define HRMR(k1,k2,k3,k4) &ht RGUI k1  &ht RCTRL k2  &ht RALT k3  &ht RSHFT k4
 
-&mt {
-  //  flavor = "tap-preferred";
-   // tapping_term_ms = <200>;
-};
+#define ________ &trans
 
-/ { 
+&caps_word {
```
Comment (petejohanson) [inline app/boards/shields/cradio/cradio.keymap:14]: Seems odd to override the existing defaults for just one keymap. Not sure I agree with this addition.

**Review by petejohanson — COMMENTED** [review]
A few other thoughts.

File: app/boards/shields/cradio/cradio.keymap (hunk 2 above)
Comment (filterpaper) [reply app/boards/shields/cradio/cradio.keymap]: Removed `&trans` substitution macro.

File: app/boards/shields/cradio/cradio.keymap (hunk 3 above)
Comment (filterpaper) [reply app/boards/shields/cradio/cradio.keymap]: Overwrite removed.

**Comment by petejohanson** [comment]
@filterpaper can you please rebase onto latest `main` for the docs fixes? Once checks pass, I'm good with this.

**Comment by filterpaper** [comment]
> @filterpaper can you please rebase onto latest `main` for the docs fixes? Once checks pass, I'm good with this.

Rebased as requested.

**Review by petejohanson — APPROVED** [review]

**Pull request merged**